pkg/payload/precondition/clusterversion/upgradeable_test.go

```diff
@@ -114,6 +136,7 @@ func TestUpgradeableRun(t *testing.T) {
 				Spec:       configv1.ClusterVersionSpec{},
 				Status: configv1.ClusterVersionStatus{
 					History: []configv1.UpdateHistory{},
+					Desired: configv1.Release{Version: tc.desiredVersionInCV},
 				},
 			}
 			if len(tc.currVersion) > 0 {
```

Comment:
The line below this appends a `Completed` history entry.  I think we want to expand the test-case structure so we can declare whether each test-case is mid-update or not, so we can get both `Progressing=True` and `Progressing=False` cases to exercise the new check?

Reply:
The field `versionPartiallyUpgraded` in the case is to claim if the cluster is in the middle of a upgrade.
`Progressing=False` is also added in the new patch.